.gitignore

```diff
@@ -1,6 +1,6 @@
 /.bundle/
 /.yardoc
-/Gemfile.lock
+*emfile.lock
```

Comment:
I assume appraisal generates something like `/rails-4.1.gemfile.lock`?

Could we tighten the matching up here?